servers.json

```diff
@@ -1655,5 +1655,12 @@
 	"addresses": [
 	    "techcraftmc.com"
 	]
+    },
+    {
+	"id": "megaplanet",
+	"name": "MEGA Planet",
+	"addresses": [
+	    "mc.megaplanet.net"
```

Comment:
Please look over the checklist.

Reply:
Ok, I have modified the address that was wrong.